api/src/pcapi/core/offers/api.py

```diff
@@ -1664,9 +1666,9 @@ def move_event_offer(
         price_category_label: _get_or_create_same_price_category_label(destination_venue, price_category_label)
         for price_category_label in original_price_category_labels
     }
-
     with transaction():
         offer.venue = destination_venue
+        offer.offererAddress = destination_venue.offererAddress
```

Comment:
Je ne vois pas trop le besoin d'avoir `joinedload` les `offererAddress` dans `edit_offer_venue` (BO blueprint), car tu pouvais simplement associer les id, le contenu n'ayant pas d'importance ici : 
```suggestion
        offer.offererAddressId = destination_venue.offererAddressId
```